client/package.json

```diff
@@ -25,7 +25,6 @@
     "is-inside-container": "^1.0.0",
     "lucide-react": "^0.395.0",
     "next": "^14.2.4",
-    "next-themes": "^0.3.0",
```

Comment:
don't delete this please

Reply:
I didnt do about this part, might due to the merge.
But anyway, I already add this into the branch and commit again